adds BarrelRacingPath to Auto Subsystem

zzberries opened the pull request.

zzberries committed:
Merge pull request #1 from Team1100/master

merge with main repository

zzberries committed:
adds BarrelRacingPath command for Auto Subsystem

BarrelRacingPath command changes commands SquareAuto to the PID
commands.

snowpuppy commented on src/main/java/frc/robot/commands/Auto/BarrelRacingPath.java:
```diff
@@ -0,0 +1,85 @@
+    double speed = 0.5;
+    addCommands(
+      //new PIDDriveDistance(144.0,true),
+      /**new PIDTurnAngle(90, true));
```
Why is all of this commented out?

snowpuppy commented on src/main/java/frc/robot/commands/Auto/DriveSquareAuto.java:
```diff
@@ -22,14 +24,22 @@ public DriveSquareAuto() {
     // addCommands(new FooCommand(), new BarCommand());
-    addCommands(new DriveDistance(distance, speed,true),
+    addCommands(/**new DriveDistance(distance, speed,true),
                 new TurnAngle(angle, speed, true),
```
Please remove older commented out code. We can use git commit history to refer to old changes if we want to.

snowpuppy commented on src/main/java/frc/robot/commands/Drive/PIDDriveDistance.java:
```diff
@@ -70,7 +70,7 @@ public static void registerWithTestingDashboard() {
     super.initialize();
     if (m_parameterized) {
-      m_setpoint = TestingDashboard.getInstance().getNumber(m_drive, "DistanceToTravelInInches");
+      //m_setpoint = TestingDashboard.getInstance().getNumber(m_drive, "DistanceToTravelInInches");
```
Please don't add commented out code.

FErfvv committed:
Update BarrelRacingPath.java

This commit uncomments part of the command and changes commands to PID commands

FErfvv committed:
reverts changes

FErfvv committed:
fixes parameterization bug

FErfvv committed:
fixes parameterization bug
